.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,60 @@
+name: build-with-ansible-lint
```

Comment:
Why do we need separate workflow for lint?

Reply:
Martin suggested, so ansible-lint will run only on pr's and not on merge

Reply:
Hmm what if we would move it back with `if: !github.event.pull_request.merged` or some simple check? I don't like to have two same builds even though one does a bit more.